spec/lib/report_formater/jqplot_formater_spec.rb

```diff
@@ -34,7 +38,7 @@
             ["tak", 1, system_name_4,          359],
           ],
         )
-
+        
```

Comment:
@martinpovolny this file name seems to misspelled - needs 2 t's in `formatter`.  So, `jqplot_formater_spec.rb` to `jqplot_formatter_spec.rb` - should be done in a separate PR.